lesson_00/ljmcwilliams/README.md

```diff
@@ -0,0 +1,30 @@
+## Introduction
```

Comment:
Consider including a title for your README.

Reply:
just added a title and a second image